data_generator.py

```diff
@@ -268,6 +268,9 @@ def __getitem__(self, idx):
                 # Perform data augmentation randomly
                 image_aug, kpsoi_aug = self.augmenter(image=transformed_img, keypoints=kpsoi)
 
+                # Perform a R/L augmentation randomly
```

Comment:
This is a bit wordy, so feel free to reword.

```suggestion
                # Perform a R/L augmentation randomly, applying R/L flip to the labels as well to maintain the right order 
```